### ISSUE TITLE
CELA P1_A11y_code.visualstudio.com_VisualStudioCode_GettingStartedwithPythoninVSCode_AI4D: Link  "get-pip.py" has no name set.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/vscode-docs#3584]
[“Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#CELA P1_A11y_code.visualstudio.com;#A11ySev2;#A11yTCS;#A11yMAS;#Benchmark;#ChromiumEdge;#DesktopWeb;#MAS4.1.2;#MAS1.3.1;#MAS4.2.1;#Win10;#Visual Studio Code;#CELAP1_code;#AI4D;

**Environment Details:**
Application Name: code.visualstudio.com
#URL:https://code.visualstudio.com/docs/python/python-tutorial
Windows Version: Windows10
Browser version: Microsoft Edge Version 81.0.416.60 (Official build) beta (64-bit)

**Repro Steps:**
1)Launch the application url :
code.visualstudio.com/docs/python/python-tutorial 
2)Getting Started with Python in VS Code  Page will get open.
3)Go through heading "Install a Python interpreter".
4) Launch accessibility insights for desktop and hover over link "get-pip.py".

**Actual:**
On hovering over the link "get-pip.py" there is no name set for it.

**Expected:**
A name should be set for the link so the screen reader announces it.

**User Impact:**
Screen reader users will not receive the name of the link visually present on the screen.

**Attachment link for Reference:**
![image](https://user-images.githubusercontent.com/24580570/79856267-c2615a00-83e9-11ea-8d1b-efdd5aee84b3.png)

**Comment by gregvanl** [comment]
Inline code blocks in links was problematic. Also difficult for sighted readers to see that text is a link.

**Comment by Yashi1201** [comment]
Issue is not repro in env https://code.visualstudio.com/docs/python/python-tutorial#_install-a-python-interpreter
Please refer attachement
![3584_Issue Fixed](https://user-images.githubusercontent.com/93374043/216595747-3670ea8e-4d9e-4a4a-a709-1a97d2f86bbc.png)

Hence, closing the bug.
**GithubTags**:#Closed;
